kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java

```diff
@@ -14,6 +14,13 @@ public class SchemaRegistryList {
 
     private final SelenideElement schemaButton = $(By.xpath("//*[contains(text(),'Create Schema')]"));
 
+    @Step
+    public SchemaRegistryList waitUntilScreenReady(){
+        $x("//*[contains(text(),'Loading')]").shouldBe(Condition.disappear);
+        $x("//button[text()=' Create Schema']").shouldBe(Condition.visible);
```

Comment:
it's bad practice to use some strange spaces in locators
better use 'contains' in this case

Reply:
Fixed